### ISSUE TITLE
dependency (typer/click) error with v2.10.1

### ISSUE THREAD

**Issue by T4mmi** [cruft/cruft#149]
Hi, I just discover `cruft` and I can't wait to try it ... **but** ...  
```sh
$ python -m pip install cruft  # use cruft==2.10.1 to reproduce
$ cruft
Traceback (most recent call last):
  File "D:\Conda\Scripts\cruft-script.py", line 5, in <module>
    from cruft._cli import app
  File "D:\Conda\lib\site-packages\cruft\__init__.py", line 6, in <module>
    from cruft._commands import check, create, diff, link, update
  File "D:\Conda\lib\site-packages\cruft\_commands\__init__.py", line 2, in <module>
    from .check import check
  File "D:\Conda\lib\site-packages\cruft\_commands\check.py", line 5, in <module>
    import typer
  File "D:\Conda\lib\site-packages\typer\__init__.py", line 29, in <module>
    from .main import Typer as Typer
  File "D:\Conda\lib\site-packages\typer\main.py", line 11, in <module>
    from .completion import get_completion_inspect_parameters
  File "D:\Conda\lib\site-packages\typer\completion.py", line 10, in <module>
    import click._bashcomplete
ModuleNotFoundError: No module named 'click._bashcomplete'
```

It seems that there is a problem with `typer`/`click` backward compatibility : https://github.com/tiangolo/typer/issues/278  

I see that a patch has been applied in requirements : https://github.com/cruft/cruft/blob/0c6de85d974197969c0e65913857eaa36b788e5e/pyproject.toml#L14 
is there any plan to bump/release a corrected version of cruft anytime soon ?  

**meanwhile if anyone faces the same problem, just manually update `typer`, it solves the problem ` pip install -U typer`**

*I realize this ain't a real issue, but more a release request ... sorry about that  :confounded:  just wanted to log the issue/fix so anyone could find a patch easily until new release ... feel free to autoclose it !*

**Comment by sharadpattanshetti** [comment]
Thank you @T4mmi  for posting this. It is working only if we pin click==8.0.4 
Any plans to provide fix through cruft itself?

**Comment by samj1912** [comment]
Should be fixed with 2.11.0. Please reopen if issue persists.
